content/en/01-getting-started/05-configuration.md

```diff
@@ -1,76 +1,52 @@
+ - `wp-config.common.php` for constants common to all environments (it's part of the Git repository).
 
-VersionPress first looks for a value in the `vpconfig.neon` file and falls back to the `vpconfig.defaults.neon` file if it doesn't find one. The `vpconfig.neon` file is meant to be edited, the `vpconfig.defaults.neon` file is meant to be read-only.
-
-The files are in the [NEON file format](http://ne-on.org/) which is pretty simple and similar to YAML. You can manually copy and paste lines from the `vpconfig.defaults.neon` file to the `vpconfig.neon` file to change VersionPress configuration.
-
-<div class="note">
-  <p>As mentioned above, always modify the `vpconfig.neon` file, not the defaults file.</p>
-</div>
 
+It is recommended to use the WP-CLI command, however, it is possible to add/change the constants manually. Just be sure they are in the right file.
 
 ### WP-CLI command
 
 As an alternative to manual editing, you can use the **`vp config`** WP-CLI command like this:
 
-    wp vp config <option> <value>
+    wp vp config <constant> <value>
     
     # e.g., set custom Git path:
-    wp vp config git-binary /custom/path/to/git
+    wp vp config VP_GIT_BINARY '/custom/path/to/git'
 
 This will work if you have [WP-CLI installed](https://github.com/wp-cli/wp-cli/wiki/Alternative-Install-Methods) and VersionPress plugin activated. Run `wp help vp config` to see the help.
 
 
 ## Config options
 
-This section lists all the supported config options.
+This section lists all the supported constants.
 
-### git-binary
+### VP\_GIT\_BINARY
 
-*Default: `git`*
+*Default: `git`*  
+*Configuration file: `wp-config.php`*
 
 By default, VersionPress calls just `git` which leaves the path resolution up to the operating system. That might be problematic on some server configurations which use different `PATH` for different users (the web server user might not be the same user under which you are logged in), there might be some `PATH` caching involved, etc. If VersionPress cannot detect Git for some reason, use this option.
 
-Example:
-
-    # regarding paths, please read notes below!
-    
-    git-binary: /path/to/git            # Linux / Mac OS
-    git-binary: 'C:\path to\git.exe'    # Windows
-
+### VP\_VPDB\_DIR
 
-Important notes about paths:
+*Default: `WP_CONTENT_DIR . '/vpdb'`*  
+*Configuration file: `wp-config.common.php`*
 
- - Putting **single quotes around the path makes it safe**, however, it's not strictly necessary if the path is simple. This is usually the case on Linux / MacOS where the path typically looks like `/usr/bin/git` or similar.
- - **On Windows**, the Git binary is usually `C:\Program Files (x86)\Git\bin\git.exe`. Here, **quoting is required** as `(...)` has a special meaning in NEON. Here are examples of valid and invalid paths on Windows:
+By default, VersionPress saves all its content into the `vpdb` directory under `WP_CONTENT_DIR`. You can change it by setting this constant.
```

Comment:
People often want to have `vpdb` outside of web root. We should state that it's not possible (the `vpdb` folder must be inside Git / project root).